src/routes/api/subscription/api-dummy-subscription-post.ts

```diff
@@ -0,0 +1,34 @@
+import { Subscription } from "models/subscription";
+import { saveConfiguredAppProperties } from "utils/app-properties-utils";
+
+/**
+ * This endpoint is a copy of the github-configuration-post.
+ * This is used by the Pollinator tests for checking the config state endpoint.
+ *
+ * This avoids all the token checks,
+ * Adds a Dummy Subscription whose installationId is `12345`
+ * Updates the config state to true.
+ *
+ * @param req
+ * @param res
+ * @constructor
+ */
+export const ApiDummySubscriptionPost = async (req, res) => {
+	const jiraHost = req.params.jiraHost as string;
+
+	try {
+		// Create a dummy Subscription
+		await Subscription.install({
```

Comment:
Actually, do we even need this line? Maybe what we need is just to use api to `await saveConfiguredAppProperties(jiraHost, req.log, true);`

Reply:
True the main thing here is the `saveConfiguredAppProperties`. 
So, I've updated the `ApiConfigurationPost` route.